katalog/pomerium/secrets/pomerium.example.env

```
@@ -4,3 +4,5 @@ COOKIE_SECRET=super-secret-cookie
 IDP_CLIENT_SECRET=super-secret-idp
 # SHARED_SECRET is obtained with  `head -c32 /dev/urandom | base64` see https://www.pomerium.io/reference/#shared-secret
 SHARED_SECRET=super-secret-shared
+
```

This env var is missing:
```suggestion
DATABROKER_STORAGE_TYPE=postgres
```

---

It is in "config.example.env"